### PR TITLE
Update and add links to sidebar footer

### PR DESCRIPTION
**Pull request by NatanRolnik**
* **Docs** point to the right link
* Removed the **Downloads** link, replacing it with the **GitHub** link
* Added more relevant links in the more section

**Before:**
captura de tela 2017-02-14 as 12 49 00

**After:**
captura de tela 2017-02-14 as 12 49 15

@hramos you might like this one :)

### CODE REVIEW

File: src/components/Sidebar/FooterMenu.react.js [hunk 1]
```diff
@@ -43,8 +43,10 @@ export default class FooterMenu extends React.Component {
           onExternalClick={() => this.setState({ show: false })}>
           <div className={styles.popup}>
             <a href={`${mountPath}logout`}>Log Out <span className={styles.emoji}>👋</span></a>
-            <a target='_blank' href='https://www.parse.com/docs/server/guide'>Server Guide <span className={styles.emoji}>📚</span></a>
-            <a target='_blank' href='https://www.parse.com/help'>Help <span className={styles.emoji}>💊</span></a>
+            <a target='_blank' href='https://github.com/ParsePlatform/parse-server/wiki'>Server Wiki <span className={styles.emoji}>📚</span></a>
```
Comment (hramos) [inline src/components/Sidebar/FooterMenu.react.js]: Can  you link to parseplatform.github.io/docs/parse-server/guide instead? The wiki is no longer hosting the guide.

**Comment by hramos** [comment]
Looks great!

File: src/components/Sidebar/FooterMenu.react.js (hunk 1 above)
Comment (NatanRolnik) [inline src/components/Sidebar/FooterMenu.react.js]: Sure! Will update.

**Comment by sepatus** [comment]
Good

**Pull request merged**